src/main/java/no/unit/nva/model/instancetypes/ReportContent.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Pages;
+import no.unit.nva.model.pages.Range;
+import nva.commons.utils.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+
+import static java.util.Objects.nonNull;
+
+public abstract class ReportContent implements PublicationInstance {
+    public static final String SETTING_PEER_REVIEWED_FALSE =
+            "Setting peerReviewed to false as {} is assumed to not be peer-reviewed";
+    protected static final Logger logger = LoggerFactory.getLogger(ReportContent.class);
+
+    private Pages pages;
+    private boolean peerReviewed;
+
+    /**
+     * This constructor ensures that the peerReviewed value is always false.
+     *
+     * @param pages         the Pages of the PublicationInstance.
+     * @param peerReviewed  the value is always ignored.
+     * @throws InvalidPageTypeException if the type of Pages is incompatible with the PublicationInstance type.
+     */
+    @SuppressWarnings("PMD.ConstructorCallsOverridableMethod")
+    public ReportContent(Pages pages, boolean peerReviewed) throws InvalidPageTypeException {
+        super();
+        setPages(pages);
+        if (peerReviewed) {
```

Comment:
wouldn't be simple that isPeerReviewed returns always false and one does not pass the argument in the constructor at all? setPeerReviewed could simple do nothing.

Reply:
Should I change this @brinxmat ?

Reply:
c.f. https://github.com/BIBSYSDEV/nva-datamodel-java/blob/develop/src/main/java/no/unit/nva/model/instancetypes/JournalNonPeerReviewedContent.java